src/functions-reference/array_operations.Rmd

```diff
@@ -179,6 +179,24 @@ The squared Euclidean distance between x and y
 `real` **`squared_distance`**`(row_vector x, row_vector y)`<br>\newline
 The Euclidean distance between x and y
 
+### Quantile
+
+Produces sample quantiles corresponding to the given probabilities.
+The smallest observation corresponds to a probability of 0 and the largest
+to a probability of 1.
```

Comment:
I think it would be useful to add the blurb from the C++ docs:

```
Implements algorithm 7 from Hyndman, R. J. and Fan, Y.,
Sample quantiles in Statistical Packages (R's default quantile function)
```

That way if someone is curious about any of the weirdnesses, they can check there.

(Things like this always returns a real even if the input is an integer might catch someone off guard)